.changeset/good-mirrors-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Add concurrency to asset generation
```

Comment:
I think it would be great to expand this changeset, by mentioning the benchmarks on your website :)

Reply:
My tests are not exactly scientific, so I'd rather avoid misleading users. I can expand on what it does, though